styles/widgets/common/multiView.less

```diff
@@ -2,7 +2,10 @@
     overflow: hidden;
     width: 100%;
     height: 100%;
-    .touch-action(pinch-zoom);
+
+    &.dx-multiview-touch-action {
+        .touch-action(pinch-zoom);
```

Comment:
May be it is better to use `-ms-touch-action: pinch-zoom` in css instead of js checking